test/faker/name_test.exs

```diff
@@ -1,27 +1,31 @@
-defmodule NameTest do
+defmodule Faker.NameTest do
   use ExUnit.Case, async: true
 
+  doctest Faker.Name.En
+  doctest Faker.Name.Es
+  doctest Faker.Name.PtBr
+
   test "first_name/0" do
-    assert is_binary(Faker.Name.first_name)
+    assert is_binary(Faker.Name.first_name())
```

Comment:
I forgot to convert these binary tests to doctests and just declare `doctest Faker.Name`. I'll do that tomorrow 👍